src/AspNetCoreModuleV2/CommonLib/PollingAppOfflineApplication.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "PollingAppOfflineApplication.h"
+
+#include <experimental/filesystem>
+#include "SRWExclusiveLock.h"
+#include "HandleWrapper.h"
+
+HRESULT PollingAppOfflineApplication::CreateHandler(IHttpContext* pHttpContext, IREQUEST_HANDLER** pRequestHandler)
+{
+    try
+    {
+        *pRequestHandler = new PollingAppOfflineHandler(pHttpContext, m_strAppOfflineContent);
+    }
+    CATCH_RETURN();
+
+    return S_OK;
+}
+
+APPLICATION_STATUS PollingAppOfflineApplication::QueryStatus()
+{
+    if (AppOfflineExists())
+    {
+        return RUNNING;
+    }
+
+    return OFFLINE;
```

Comment:
This return value seems the opposite of what it actually is. Isn't this really app_offline being removed?